src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/ItemManager.cs

```diff
@@ -509,7 +509,8 @@ private static ImmutableArray<CompletionItemWithHighlight> GetHighlightedList(Li
             ImmutableArray<CompletionFilterWithState> filters)
         {
             // See which filters might be enabled based on the typed code
-            var textFilteredFilters = filteredList.SelectMany(n => n.VSCompletionItem.Filters).ToHashSet();
+            var _ = PooledHashSet<CompletionFilter>.GetInstance(out var textFilteredFilters);
+            textFilteredFilters.AddRange(filteredList.SelectMany(n => n.VSCompletionItem.Filters));
```

Comment:
😕 Why is this changing?

Reply:
`ToHashSet` is not available on netstardard2.0

Reply:
Ah, the direct translation would be `new HashSet<CompletionFilter>(expr)`.